bundles/org.openhab.binding.comfoair/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,1596 @@
+		<description>State of the chimney control</description>
+		<category>Switch</category>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="bypass_state" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Bypass State</label>
+		<description>State of the bypass (ON = open / OFF = closed)</description>
+		<category>Switch</category>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="ewt_state" advanced="true">
+		<item-type>Switch</item-type>
+		<label>EWT State</label>
+		<description>State of the EWT valve (ON = open / OFF = closed)</description>
+		<category>Switch</category>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="heater_state" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Heater State</label>
+		<description>State of the heater</description>
+		<category>Switch</category>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="v_control_state" advanced="true">
+		<item-type>Switch</item-type>
+		<label>0-10V Control State</label>
+		<description>State of the 0-10V control</description>
+		<category>Switch</category>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="frost_state" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Antifrost State</label>
+		<description>State of the antifrost control</description>
+		<category>Switch</category>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="cookerhood_state" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Cookerhood State</label>
+		<description>State of the cookerhood control</description>
+		<category>Switch</category>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="enthalpy_state" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Enthalpy State</label>
+		<description>State of the enthalpy module</description>
+		<category>Switch</category>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="ewt_temperature" advanced="true">
+		<item-type>Number:Temperature</item-type>
+		<label>EWT Temperature</label>
+		<description>Temperature of geothermal heat exchanger sensor</description>
+		<category>Temperature</category>
+		<state readOnly="true" pattern="%.1f %unit%" />
+	</channel-type>
+
+	<channel-type id="heater_temperature" advanced="true">
+		<item-type>Number:Temperature</item-type>
+		<label>Heater Temperature</label>
+		<description>Temperature of heater sensor</description>
+		<category>Temperature</category>
+		<state readOnly="true" pattern="%.1f %unit%" />
+	</channel-type>
+
+	<channel-type id="cookerhood_temperature" advanced="true">
+		<item-type>Number:Temperature</item-type>
+		<label>Cookerhood Temperature</label>
+		<description>Temperature of cookerhood sensor</description>
+		<category>Temperature</category>
+		<state readOnly="true" pattern="%.1f %unit%" />
+	</channel-type>
+
+	<channel-type id="level0_time" advanced="true">
+		<item-type>Number</item-type>
```

Comment:
I'm not sure if this is really useful here. The device always returns these values in hours, therefore a transformation to smaller units (minutes, seconds) is pretty useless (or even suggesting a wrong level of accuracy). Pretty much the same accounts for the transformation to bigger values (days, wekks, ...) since these do usually not represent a continouos operating state over such a long duration but counting overall operating hours on that state (therefore also the previous 'operating hours' terminology).
I'm okay with changing this to `Number:Time`, if it makes sense within the OH guidelines, however I would be interested what other @openhab/add-ons-maintainers think.

Reply:
All readonly channels should use units if possible.

Reply:
Alright, will do then. I also noticed your comment in [#1470 (comment)](https://github.com/openhab/openhab-core/pull/1470#issuecomment-634393163). With an appropriate duration format, UoM would make definitly sense here.